repobee_sanitizer/_sanitize_repo.py

```diff
@@ -136,7 +140,11 @@ def _git_commit_on_branch(repo_root: pathlib.Path, target_branch: str):
     repo = git.Repo(str(repo_root))
     repo.git.symbolic_ref("HEAD", f"refs/heads/{target_branch}")
     repo.git.add(".", "--force")
-    repo.git.commit("-m", "'Sanitize files'")
+    try:
+        repo.git.commit("-m", "'Sanitize files'")
+    except git.GitCommandError as exc:
+        if "nothing to commit, working tree clean" in exc.__str__():
```

Comment:
In general, we don't invoke dunder (double underscore) methods directly.

```suggestion
        if "nothing to commit, working tree clean" in str(exc):
```

Reply:
Now I'm thinking that perhaps we don't need this `if` statement. We currently aren't aware that there can be any other form of failure here, so we should probably just put in an assert instead.
```suggestion
        assert "nothing to commit, working tree clean" in str(exc)
```

Thoughts?

Reply:
I have no idea why str() flew above my head for that one, but its fixed now :)